src/renderer/components/mixins.scss

```diff
@@ -59,20 +59,3 @@
     @content; // css-modules (*.module.scss)
   }
 }
-
-// Makes custom @font-family available at earlier stages.
-// Element must exist in DOM as soon as possible to initiate preloading.
-@mixin font-preload($fontFamily) {
```

Comment:
Seems like overkill since can serve only single element and require duplicating exported font name (via `fonts.scss`). More simple solution to define early-access font directly via `style="font-family: %ExportedNameInFontsCss%"` _(see: `template.html`)_